dotcom-rendering/src/model/enhance-tweets.ts

```diff
@@ -0,0 +1,33 @@
+const removeTweetClass = (elements: CAPIElement[]): CAPIElement[] => {
+	const enhanced: CAPIElement[] = [];
+	elements.forEach((element) => {
+		switch (element._type) {
+			case 'model.dotcomrendering.pageElements.TweetBlockElement':
+				// Parse the embed block element, find the iframe and add
+				// the alt text as a title attribute if it exists
+				// https://dequeuniversity.com/tips/provide-iframe-titles
+				const withoutTweetClass = element.html.replace(
+					'twitter-tweet',
+					'nojs-tweet',
+				);
+				enhanced.push({
+					...element,
+					...{
+						html: withoutTweetClass,
+					},
+				});
```

Comment:
not a biggy, but maybe easier to read
```suggestion ts
				enhanced.push({
					...element,
					html: withoutTweetClass,
				});
```